jazzy/distribution.yaml

```diff
@@ -991,6 +991,25 @@ repositories:
       url: https://github.com/carologistics/clips_vendor.git
       version: main
     status: maintained
+  cob_common:
+    doc:
+      type: git
+      url: https://github.com/4am-robotics/cob_common.git
+      version: foxy
+    release:
+      packages:
+      - cob_actions
+      - cob_msgs
+      - cob_srvs
+      tags:
+        release: release/jazzy/{package}/{version}
+      url: https://github.com/4am-robotics/cob_common-release.git
```

Comment:
This is allowed for `jazzy`, but please see https://github.com/ros/rosdistro/pull/43654#pullrequestreview-2458666540 .  It is usually a good idea to move this release repository to `ros2-gbp`, and then to use that release repository for everything.